big-practice/src/scripts/views/customer-view.js

```diff
@@ -1,27 +1,38 @@
-import imgDot from '../../assets/images/dot.png';
+import imgDot from '../../assets/images/dot.png'
 
 class CustomerView {
   constructor() {
-    this.table = document.querySelector('.table-customer-body');
-    this.iconAddCustomer = document.querySelector('.icon-add-modal');
-    this.modalCustomer = document.querySelector('.modal-customer');
-    this.iconCancel = document.querySelector('.icon-cancel');
-    this.btnCancel = document.querySelector('.btn-secondary');
-    this.formCustomer = document.querySelector('.form-customer');
+    this.table = document.querySelector('.table-customer-body')
+    this.iconAddCustomer = document.querySelector('.icon-add-modal')
+    this.modalCustomer = document.querySelector('.modal-customer')
+    this.iconCancel = document.querySelector('.icon-cancel')
+    this.btnCancel = document.querySelector('.btn-secondary')
+    this.formCustomer = document.querySelector('.form-customer')
+    this.inputTitle = document.querySelectorAll('.input-title')
+    this.countrySelect = document.getElementById('country')
+    this.inputFields = this.modalCustomer.querySelectorAll('input');
+    this.errorMessages = this.modalCustomer.querySelectorAll('.mess-invalid-form')
 
     const showCustomerModal = () => {
```

Comment:
Should move these codes outside the constructor.
Constructor should be used only to initialize object's state / fields. We should delegate the responsibility of other things to other methods.